common/sonicv2connector.h

```diff
@@ -91,6 +91,9 @@ class SonicV2Connector_Native
 
         def keys(self, *args, **kwargs):
             return list(super(SonicV2Connector, self).keys(*args, **kwargs))
+
+        def del_key(self, db_name, key, blocking=False):
+            return super(SonicV2Connector, self).del(db_name, key, blocking)
```

Comment:
This seems to be a new method, why it is added?

Reply:
Thanks for noticing it, I thought I removed it. I am working on the db migration - rename the table entries when migrating to the updated version with advanced instead of warm table entries. I wasn't sure if I should use redis rename command or get-set-del for renaming.
Rename is not supported yet so I thought I would use get-set-del sequence, however this shouldn't be part of the PR at the moment as I am not done with it.
Again - thanks for noticing.